utils/read_xml.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
We currently have a method called "writeMavenSettingsFile" in utils. See if you can use that  . It is for inserting github username and password to settings.xml

Reply:
Not applicable , I thought it is UI PR.

Reply:
@sshveta it is in UI repo, not here, and it is written in JS, not in Python. Don't think I can use that code here :-)